kola/tests/coretest/core.go

```diff
@@ -170,6 +195,46 @@ func TestDockerPing() error {
 	}
 }
 
+func TestCrioEcho() error {
```

Comment:
Note: `podman` doesn't test `cri-o`. We will need to get @mrunalp or @runcom to give us proper instructions for using `cri-o` without kube and without dropping down to `runc`.

Reply:
I suggest using crictl like how we do in our integration tests. For e.g. https://github.com/kubernetes-incubator/cri-o/blob/master/test/network.bats#L16